code/src/main/webapp/CSS/style.css

```diff
@@ -1,124 +1,307 @@
 body, html {
+  color: #222;
   height: 100%;
   margin: 0;
-  color: #222;
 }
 
 h1, p, a {
+  color:#025696;
+  font-family: Muli;
+  text-shadow: 0 0 18px rgba(255, 255, 255, 1);
   margin: 0;
 }
 
 div {
   position: fixed;
-  width: 100%;
   text-align: center;
+  width: 100%;
 }
 
-#wrapper-background {
-  height:100%;
+.fade-in {
+  animation: fadeIn ease 1s;
+  -webkit-animation: fadeIn ease 1s;
+  -moz-animation: fadeIn ease 1s;
+  -o-animation: fadeIn ease 1s;
+  -ms-animation: fadeIn ease 1s;
+}
+.fade-out {
+  animation: fadeOut ease 1s;
+  -webkit-animation: fadeOut ease 1s;
+  -moz-animation: fadeOut ease 1s;
+  -o-animation: fadeOut ease 1s;
+  -ms-animation: fadeOut ease 1s;
+}
+
+.FAQ {
+  width: 80%;
+}
+
+.FAQ-answers {
+  font-size: 1vw;
+  width: 60%;
+}
+
+.FAQ-questions {
+  font-size: 2vw;
+  width: 50%;
+}
+
+.general-background {
+  background-image: url("images/background-login-blue.svg");
   background-position: center;
   background-repeat: no-repeat;
-  background-size: 120% 100%;
-  transition: 1s background linear;
+  background-size: 100% 100%;
+  height: 100%;
+}
+
+.hidden {
+  display: none;
 }
 
 .landing-background {
-  background-image: url("images/background-landing.svg");
+  background-image: url("images/background-landing-blue.svg");
+}
+
+.landing-background #blue-buttons-to-procede {
+  top: 60%;
+}
+
+.login-background #blue-buttons-to-procede {
+  top: 60%;
 }
 
 .landing-background .line {
   height: 0;
   width: 0;
 }
 
-.login-background {
-  background-image: url("images/background-login.svg");
+.line {
+  background: #3293d3;
 }
 
-.line {
-  background: #a3c4f7;
+.line-horizontal {
+  height: 1rem;
+  transition: 1s width linear;
+  width: 100%;
 }
 
 .line-vertical {
-  width: 1rem;
   height: 100%;
   transition: 1s height linear;
+  width: 1rem;
 }
 
-.line-horizontal {
-  height: 1rem;
-  width: 100%;
-  transition: 1s width linear;
+.login {
+  top: 30%;
 }
 
-#line-vertical-right {
-  left: 12%;
-  transition-delay: 1s
+.login-background {
+  background-image: url("images/background-login-blue.svg");
 }
 
-#line-vertical-left {
-  left: 8%;
-  transition-delay: 1.2s
+#blue-buttons-to-procede button {
+  background: #025696;
+  border: 8px solid #025696;
+  border-radius: 5rem;
+  color: #ffffff;
+  font-family: Muli;
+  font-size: 3vw;
+  margin: 2rem;
+  padding: 2rem;
+  text-decoration: none;
+  top: 60%;
 }
 
-#line-horizontal-top {
-  top: 80%;
-  transition-delay: 1s
+#blue-buttons-to-procede button:hover {
+  box-shadow: 0 0 2rem #8bccf4;
```

Comment:
This is pretty subtle given the background changes. Perhaps make the box shaddow more of a transparent grey, so that it overlays nicely on the background regardless of the background color.